### ISSUE TITLE
I can not access with external ip.

### ISSUE THREAD

**Issue by lgdelai** [traccar/traccar#1873]
Hello.

I own a public ip, my computer has installed Xampp and I can access it normally using the address for example: "http://123.123.123.123/xampp"

But I can not access the traccar, "http://123.123.123.123:8081"

And the android phone can not connect to the server using the address "123.123.123.123"

Is there anything else I need to do for work? how to release a door for example?.

**Comment by tananaev** [comment]
Following page might help:

https://www.traccar.org/troubleshooting/

**Comment by lgdelai** [comment]
Thank you for your help.

8082 The door was locked on my server, then to take the test I changed the port to 80 in the xml file. Now I can access the web via system.

But still I can not identify the two smartphones that I set.

I take a doubt, the Android client after installing the "Server port" was 5055, if I leave so get shipping error, if I move to the same port that I am using on the server, in this case "80" the error does not appear but the devices are not identified on the server.

After all, On Android Device, I leave in the "Server port" the 5055 port and release it.
Or it will be the same as I am using on the server, 8082, "in my test 80" etc.
